masterlist.yaml

```diff
@@ -22290,9 +22317,14 @@ plugins:
     clean:
       - crc: 0xFC69BF72
         util: 'SSEEdit v4.0.3'
+      - crc: 0x3297A568
+        util: 'SSEEdit v4.0.3'
```

Comment:
I think this is supposed to be for *ClockworkSorting_Ing_CCCurios_CACO.esp*, which already has this cleaning data.

Reply:
Yup. My bad.